cmd/webhook/app/webhook.go

```diff
@@ -39,7 +38,7 @@ var validationHook handlers.ValidatingAdmissionHook = handlers.NewRegistryBacked
 var mutationHook handlers.MutatingAdmissionHook = handlers.NewSchemeBackedDefaulter(logs.Log, webhook.Scheme)
 var conversionHook handlers.ConversionHook = handlers.NewSchemeBackedConverter(logs.Log, webhook.Scheme)
 
-func NewServer(opts options.WebhookOptions, stopCh <-chan struct{}) (*server.Server, error) {
+func NewServer(opts *options.WebhookOptions, stopCh <-chan struct{}) (*server.Server, error) {
```

Comment:
Why change this to accept a pointer? The options are required, and I'm not sure if passing in `nil` options makes sense (and from what I can tell, will probably cause a panic..)